poglink/cogs/rates.py

```diff
@@ -108,71 +113,73 @@ async def send_embed(self, description, url):
             logger.info("Announcement channel detected: Publishing message")
             await message.publish()
 
+    async def compare_posted_rates(self, webpage_url, output_path):
+        # get current rates from ARK Web API
+        logger.info(f"Retrieving current rates at {webpage_url}")
+        try:
+            rates = await self.get_current_rates(webpage_url)
+        except ValueError as e:
+            raise RatesProcessError(e) from e
+        except Exception as e:
+            raise RatesFetchError(e) from e
```

Comment:
Using these custom exceptions for both testing purposes and for better readability / error handling possibility.